src/ansys/geometry/core/math/bbox.py

```diff
@@ -206,6 +207,42 @@ def __eq__(self, other: "BoundingBox2D") -> bool:
             and self.y_max == other.y_max
         )
 
+    @check_input_types
     def __ne__(self, other: "BoundingBox2D") -> bool:
         """Not equals operator for the ``BoundingBox2D`` class."""
         return not self == other
+
+    @check_input_types
+    def __and__(self, other: "BoundingBox2D") -> Union[None, "BoundingBox2D"]:
+        """Bitwise and operator for the ``BoundingBox2D`` class."""
+        intersect, minX, maxX = self.intersect_interval(
```

Comment:
PEP8 - please use snake case. camelCase variables are not accepted. This applies everywhere